SparkPost/2web2ui
# TR-1088: Improve domain setup UX for auto verified accounts

### What Changed

For accounts with the auto verify domains option enabled, the re-verify buttons have been removed from the sending domain edit page and a banner was added only for "verified" domains.

<img width="1387" alt="screen shot 2019-03-01 at 3 30 25 pm" src="https://user-images.githubusercontent.com/1335605/53664629-fe11f100-3c36-11e9-94b2-33fbcd2ee0c8.png">

_Please note that domain status is also displayed on the list page and on the subaccount sending domains tab.  No changes are needed.  They both display ready for values when a domain is verified._ 

In addition, removed "Ready For" tags from edit page for sending domains that are "pending" or "blocked".  This was not in the original scope of this ticket, but is a bug that needed to be fixed.

<img width="1389" alt="screen shot 2019-03-01 at 1 02 56 pm" src="https://user-images.githubusercontent.com/1335605/53660784-b25a4a00-3c2c-11e9-8246-7d9324c4e61a.png">

### How To Test

_Please note that we cannot fully test this ticket until AC-619 is released._

1. Create a free account, create a sending domain, and confirm that it is unverified.  (for sanity, click the verify buttons to confirm they fail)
2. Upgrade the account to a paid plan and enable the account option ([see instructions](https://jira.int.messagesystems.com/browse/TR-1088?focusedCommentId=488868&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-488868)).  Then confirm that the auto verify stamp icons are displayed and the verify buttons are not displayed.
